crates/sp-domains/src/lib.rs

```diff
@@ -187,13 +241,38 @@ pub struct BundleHeader<Number, Hash> {
     pub slot_number: u64,
     /// The merkle root of the extrinsics.
     pub extrinsics_root: H256,
+    /// Solution of the bundle election.
+    pub bundle_solution: BundleSolution<DomainHash>,
+    /// Signature of the bundle.
+    pub signature: ExecutorSignature,
 }
```

Comment:
Why not nesting `BundleHeader` in `SignedBundleHeader` or `SealedBundleHeader`? This is a strange approach and the only reason for it I can come up with is such that some of existing code doesn't need to change.

Reply:
I hesitated to nest the `BundleHeader` because, in most cases, we primarily work with the `SealBundleHeader` instead of the `BundleHeader`. To facilitate working with the `SealBundleHeader` more conveniently, I opted to make the default assumption and enable direct usage of `header.foo` instead of `sealed_header.header.foo`.

The naming you suggested looks good to me though, I can rename them to `BundleHeader` and `SealedBundleHeader`.

Reply:
The concern of missing a field during the refactoring makes sense, I'm not super satisfied with `let header = sealed_header.header;`, but happy to take this suggestion.